pyscriptjs/tests/integration/test_01_basic.py

```diff
@@ -333,6 +357,10 @@ def test_getPySrc_returns_source_code(self):
             == 'print("hello world!")\n'
         )
 
+    @pytest.mark.skip(
+        reason="BROKEN: Error: The interpreter 'py' was not found. Available "
+        "interpreters are: 'py-script', 'pyodide'."
```

Comment:
IIRC this has been fixed with latest pyscript so I am not sure why this test would break.

Reply:
It was still showing up in the tests. We can double check for sure